addons/l10n_pl_jpk/views/account_move_views.xml

```diff
@@ -10,7 +10,7 @@
                 <field name="l10n_pl_delivery_date" string="Delivery Date" attrs="{'invisible': [('l10n_pl_show_delivery_date', '=', False)]}"/>
             </field>
             <field name="qr_code_method" position="after">
-                <field name="l10n_pl_delivery_date" string="Delivery Date"/>
+                <field name="l10n_pl_delivery_date" string="Delivery Date" attrs="{'invisible': [('country_code', '!=', 'PL')]}"/>
```

Comment:
Why don't we use `l10n_pl_show_delivery_date` here instead of the country_code then ?

Reply:
Because here, we want it always be visible.
So, the first use of it in the view, with the show_delivery_date, appears in the "main" fields of the form view (with the invoice date etc), but only if it is filled already.
It can be filled if it comes from the sale order, or they can fill it in the tab "Other Info".
That's the same generic behavior as in master, where delivery date is common for every localization. 
Some localization override this, but not Poland. It is necessary for their jpk to be able to fill it, but not everyone has to fill it (only if delivery is before the invoice).
https://github.com/odoo/odoo/blob/master/addons/account/views/account_move_views.xml#L1229 in master

Reply:
I see ! Thank you for the detailed explanation :slightly_smiling_face: